File: src/main/scala/org/ximo/finkinaction/scala/domain/Student.scala
package org.ximo.finkinaction.scala.domain

/**
  *
  *
  * @author xikl
  * @date 2019/9/8
  */
case class Student(name: String, age: Int) {

  var id: Int = _

  def this(id: Int, name: String, age: Int) {
    this(name, age)
    this.id = id
  }

}

File: src/main/java/org/ximo/finkinaction/java/domain/Student.java
package org.ximo.finkinaction.java.domain;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

/**
 * @author xikl
 * @date 2019/9/8
 */
@Data
@NoArgsConstructor
@AllArgsConstructor
public class Student {

    private Integer id;

    private String name;

    private Integer age;

    public Student(String name, Integer age) {
        this.name = name;
        this.age = age;
    }
}
